tests/doc_issues/test_collector.py

```
@@ -11,9 +11,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+from tkinter.filedialog import dialogstates
```

Ruff (0.11.9)

`tkinter.filedialog.dialogstates` imported but unused

Remove unused import: `tkinter.filedialog.dialogstates`

(F401)